### PR TITLE
[EDU-4186] feat: add CRL digital certificates

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added reference for CRL in Digital Certificates in EN and PT.

[EDU-4186]: https://aziontech.atlassian.net/browse/EDU-4186?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx [hunk 1]
```diff
@@ -117,6 +121,26 @@ For customers with contracted FIPS 140 support, the private key will be stored i
 
 ---
 
+## Certificate Revocation List (CRL)
+
+A **Certificate Revocation List (CRL)** is a list of digital certificates revoked before their expiration date. You can add it in Azion, and it works as a blocking list, stating those certificates can't be trusted. The issuing certificate authority (CA) signs this list to guarantee its security.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx:126]:
```suggestion
A **Certificate Revocation List (CRL)** is a list of digital certificates revoked before their expiration date. It works as a block list that you can add it to your account to state those certificates can't be trusted. The issuing certificate authority (CA) signs this list to guarantee its security.
```

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx [hunk 2]
```diff
@@ -117,6 +121,26 @@ For customers with contracted FIPS 140 support, the private key will be stored i
...
+
+The CRL works when associated with a [mTLS CA Certificate](/en/documentation/products/build/edge-application/domains/mtls/). Therefore, you must have mTLS enabled in the domain you want to associate it with.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx:128]:
```suggestion
The CRL works when associated with an [mTLS CA Certificate](/en/documentation/products/build/edge-application/domains/mtls/). Therefore, you must have mTLS enabled in the domain you want to associate it with.
```

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx [hunk 3]
```diff
@@ -117,6 +121,26 @@ For customers with contracted FIPS 140 support, the private key will be stored i
...
+
+The CRL is automatically validated before the creation is successful in the Azion Edge Computing Platform. The management, including creation, of a CRL is available via [Azion API](https://api.azion.com/#56c3cab2-527e-478b-8dad-1b25b2b48248).
+
+If you have multiple CRLs, you must add one list at a time, and separate each certificate with a line break.
+
+<Button href="https://api.azion.com/#56c3cab2-527e-478b-8dad-1b25b2b48248" text="create CRL via API" variant="secondary"></Button>
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx:134]:
```suggestion
<Button href="https://api.azion.com/#56c3cab2-527e-478b-8dad-1b25b2b48248" text="create CRL via API" variant="secondary" />
```

Assim fica mais clean 🤓

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx [hunk 4]
```diff
@@ -117,6 +121,26 @@ For customers with contracted FIPS 140 support, the private key will be stored i
...
+
+You can [associate a CRL with a domain](https://api.azion.com/#d3b79af9-e708-4a4f-9bc6-d2df3700c502) via API.
+
+:::note
+You can't delete a CRL currently associated with a domain.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx:139]:
```suggestion
You can't delete a CRL that is associated with a domain. You must first remove the CRL from the domain to delete it.
```

Notei que existe o delete então melhor esclarecer

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx [hunk 5]
```diff
@@ -119,6 +123,26 @@ Para clientes com suporte ao FIPS 140 contratado, a chave privada será armazena
 
 ---
 
+## Certificate Revocation List (CRL)
+
+Uma **Certificate Revocation List (CRL)** é uma lista de certificados digitais que foram revogados antes de sua data de expiração. Você pode adicionar uma CRL no Azion, e ela funciona como uma lista de bloqueio, indicando que esses certificados não são mais confiáveis. Esta lista é assinada pela autoridade certificadora (CA) emissora para garantir sua segurança.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx:128]:
```suggestion
Uma **Certificate Revocation List (CRL)** é uma lista de certificados digitais que foram revogados antes de sua data de expiração. Ela funciona como uma lista de bloqueio que você pode adicionar na sua conta, indicando que esses certificados não são confiáveis. Esta lista é assinada pela autoridade certificadora (CA) emissora para garantir sua segurança.
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx [hunk 6]
```diff
@@ -119,6 +123,26 @@ Para clientes com suporte ao FIPS 140 contratado, a chave privada será armazena
...
+
+A CRL funciona quando associada a um [Certificado CA mTLS](/pt-br/documentacao/produtos/build/edge-application/domains/mtls/). Portanto, você precisa ter mTLS habilitado no domínio ao qual deseja associar a CRL.
+
+A CRL é validada automaticamente antes da criação ser bem-sucedida na Plataforma de Edge Computing da Azion. O gerenciamento, incluindo a criação de uma CRL, está disponível através da [API Azion](https://api.azion.com/#56c3cab2-527e-478b-8dad-1b25b2b48248).
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx:132]:
```suggestion
A CRL é validada automaticamente antes de ser criada com sucesso na Plataforma de Edge Computing da Azion. O gerenciamento, incluindo a criação de uma CRL, está disponível através da [API Azion](https://api.azion.com/#56c3cab2-527e-478b-8dad-1b25b2b48248).
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx [hunk 7]
```diff
@@ -119,6 +123,26 @@ Para clientes com suporte ao FIPS 140 contratado, a chave privada será armazena
...
+
+Se você tem várias CRLs, você deve adicionar uma lista por vez e separar cada certificado por uma quebra de linha.
+
+<Button href="https://api.azion.com/#56c3cab2-527e-478b-8dad-1b25b2b48248" text="criar CRL via API" variant="secondary"></Button>
+
+Você pode [associar uma CRL a um domínio](https://api.azion.com/#d3b79af9-e708-4a4f-9bc6-d2df3700c502) via API.
+
+:::note
+Não é possível excluir uma CRL associada a um domínio.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx:141]:
```suggestion
Não é possível excluir uma CRL associada a um domínio. Você deve primeiro remover a CRL do domínio para excluí-la.
```

**Review by bru-andrade — APPROVED** [review]

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**